src/librustc_mir/borrow_check.rs

```diff
@@ -396,26 +396,31 @@ impl<'c, 'b, 'a: 'b+'c, 'gcx, 'tcx: 'a> MirBorrowckCtxt<'c, 'b, 'a, 'gcx, 'tcx>
                             ReadKind::Copy =>
                                 this.report_use_while_mutably_borrowed(
                                     context, lvalue_span, borrow),
-                            ReadKind::Borrow(bk) =>
+                            ReadKind::Borrow(bk) =>{
```

Comment:
nit: coding style in project is to put a single space between the `=>` and the `{` in a match arm